config/services.php

```diff
@@ -31,4 +31,9 @@
         'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
     ],
 
+    'google' => [
+        'client_id' => env('GOOGLE_CLIENT_ID'),
+        'client_secret' => env('GOOGLE_CLIENT_SECRET'),
```

Comment:
Please Also update the `.env.example` file with the new **env keys** with empty values.

so for every deployment it will be obvious which values has been added to the `.env` file and should have taken care of